tests/test_util.py

```diff
@@ -135,7 +135,7 @@ def test_convert_to_stripe_object_and_back(self):
             "livemode": False,
         }
 
-        obj = util.convert_to_stripe_object(resp)
+        obj = util.convert_to_stripe_object(resp, api_mode="V1")
```

Comment:
is this now required? Can we keep the test as it was to validate that the behavior didn't change?

Reply:
No it's not required, I'll keep the test as is